### PR TITLE
User story 32

### PR DESCRIPTION
**Pull request by jKwibe**
- skipped a test because I couldn't figure why it was not working.

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -31,7 +31,13 @@ def create
     end
   end
 
+  def update
```
Comment (RyN21) [inline app/controllers/orders_controller.rb:34]: Nice update method

File: app/views/admin/dashboard/index.html.erb [hunk 2]
```diff
@@ -1 +1,23 @@
-<p>Logged in as <%= @user.name %></p>
+<p>Logged in as <%= current_user.name %></p>
+<%= form_tag "/admin/dashboard", method: :get do %>
```
Comment (RyN21) [inline app/views/admin/dashboard/index.html.erb:2]: Form tag looks good

File: app/views/admin/dashboard/index.html.erb [hunk 3]
```diff
@@ -1 +1,23 @@
...
+    </select>
+
+<%= submit_tag "find" %>
+<% end %>
+<% @orders.each do |order| %>
+    <div id="order-<%= order.id %>">
+        <p>Order id: <%= order.id %></p>
+        <p>Order status: <%= order.status %></p>
+        <%= link_to "Client: #{order.user.name} ", "/admin/users/#{order.user.id}" %>
+        <p>Order was made on: <%= order.created_at.in_time_zone("Mountain Time (US & Canada)").strftime("%B %d %Y") %> </p>
```
Comment (RyN21) [inline app/views/admin/dashboard/index.html.erb:18]: I like the timezone you added

File: spec/features/admin/see_all_orders_spec.rb [hunk 4]
```diff
@@ -0,0 +1,130 @@
+
+require 'rails_helper'
```
Comment (RyN21) [inline spec/features/admin/see_all_orders_spec.rb:2]: nice test

File: app/views/admin/dashboard/index.html.erb (hunk 3 above)
Comment (sciencefixion) [reply app/views/admin/dashboard/index.html.erb:18]: Agreed! This looks awesome!

File: db/seeds.rb [hunk 5]
```diff
@@ -19,3 +21,14 @@
 #dog_shop items
 pull_toy = dog_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
 dog_bone = dog_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
+
+@admin_user = create(:user , email: 'test@test1.com',  role: 2 )
```
Comment (sciencefixion) [inline db/seeds.rb:25]: Great use of factory_bot for seeds!

**Pull request merged**